src/ert/_c_wrappers/fm/ecl/ecl_run.py

```diff
@@ -166,7 +166,7 @@ def __init__(
 
         # Dechipher the ecl_case argument.
         input_arg = ecl_case
-        (base, ext) = os.path.splitext(input_arg)
+        (_, ext) = os.path.splitext(input_arg)
```

Comment:
As you fixed this with underscore, what do you think about fixing the linting above:
`Check warning on line 30 in src/ert/_c_wrappers/fm/ecl/ecl_run.py`
`Loop control variable 'icpu' not used within the loop body. If this is intended, start the name with an underscore.`

Reply:
Its easily fixed with `for _ in range(host_numcpu):` on lne 30

Reply:
fixed